### DISCUSSION TITLE
Meeting summary (24.08.2020)

### DISCUSSION THREAD

**Discussion by szrtGreg** [DevAdventCalendar/DevAdventCalendar#364] [General] [answered]
1. Introduce the technologies/application aspects you want to deal with - All
2. If anyone has any interesting aspect related to (technology/problems/new ideas) feel free to create new discussion thread - All
3. Create detailed stats in user`s profile(Krzysiek)
4. Create generall stats page for all users (Krzysiek)
5. Integration with google callendar (Mateusz)
6. Login to application using different providers like instagram, linkedin (Grzegorz)
7. Adding some bonus questions (Kamil)
8. More extensive mailing system - sending email to users when longer inactivity or with some question hints (Mateusz)
9. Adding more questions eg. about code structure, behaviour (All)
10. Add posibility to display "code questions" with proper code formating and syntax (Grzegorz)
11. Extend interaction with user using frontend with buttons, lists, selections (Krzysiek)
12. Prepare Resources in one place (Kamil)

Have a good day!

**Comment by WTobor** [comment, marked as answer]
Thank you - it looks great ❤️

**Comment by BlackMorzan** [comment]
I think we need to add one more issue. Point 3 and 4 should be all about backend - getting and performing operations on data from database, also we need to display this data to user. That could mean entire another tab in profile and some neat way of displaying it. I think it, if possible, should be performed by frontend dev, and therefore should be another issue.

**Reply by WTobor** [reply]
If possible - yes ;p But remember, that the first stage can be a simple view without styles. And that should be the output of #366 . Of course, we can add a new task to do some magic in the frontend part. But first, let's focus on creating a simple view without magic :D
